helm/publisher/templates/scholarly-articles-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}--scholarly-articles
```

Comment:
Might be best to just call it `article-store` for now. Need to experiment with how to distinguish between two article stores in the hypermedia world.

Reply:
No opinion, will leave it generic. The important part is to distinguish it from other Deployments like a frontend container, a search or a gateway service, etc.